mission/config/remoteexec.hpp

```diff
@@ -0,0 +1,139 @@
+		FNC_CONSTRUCTOR(bis_fnc_call, ALLOW_SERVER, ENABLE_JIP)
+
+		// VANILLA MIKE FORCE SPECIFIC
+
+		// paradigm client initialisiation
+		FNC_CONSTRUCTOR(para_s_fnc_init_player, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(para_s_fnc_postinit_player, ALLOW_SERVER, DISABLE_JIP)
+
+		// paradigm utils
+		FNC_CONSTRUCTOR(para_s_fnc_rehandler, ALLOW_SERVER, ENABLE_JIP)
+		FNC_CONSTRUCTOR(para_c_fnc_show_notification, ALLOW_SERVER, DISABLE_JIP)
+
+		// rehandler functions. TODO: are these actually remote exec'd? or do we remoteExec rehandler?
+		FNC_CONSTRUCTOR(vn_mf_fnc_arsenal_trash_cleanup, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_changeteam, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_eatdrink, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_packageforslingloading, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_settrait, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_supplyrequest, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_supporttaskcreate, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_teleport, ALLOW_SERVER, DISABLE_JIP)
+
+		// change vehicles at a vehicle spawner
+		FNC_CONSTRUCTOR(vn_mf_fnc_veh_asset_handle_change_vehicle_request, ALLOW_SERVER, DISABLE_JIP)
+
+		// BRO-NATION MIKE FORCE SPECIFIC
+
+		// player site specific hold actions
+		FNC_CONSTRUCTOR(vn_mf_fnc_sites_remoteactions_reveal_radiotap, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_sites_remoteactions_reveal_scout, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_sites_remoteactions_reveal_intel, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_sites_remoteactions_destroy_task, ALLOW_SERVER, DISABLE_JIP)
+
+		// player vehicle spawner specifc hold actions
+		FNC_ALLOW_SERVER_DISABLE_JIP(vn_mf_fnc_veh_asset_bn_curator_force_recover_wrecked_vehicle, ALLOW_SERVER, DISABLE_JIP)
+		FNC_ALLOW_SERVER_DISABLE_JIP(vn_mf_fnc_veh_asset_bn_curator_force_reset_idle_vehicle, ALLOW_SERVER, DISABLE_JIP)
+		FNC_ALLOW_SERVER_DISABLE_JIP(vn_mf_fnc_veh_asset_bn_curator_lock_spawner, ALLOW_SERVER, DISABLE_JIP)
+		FNC_ALLOW_SERVER_DISABLE_JIP(vn_mf_fnc_veh_asset_bn_curator_unlock_spawner, ALLOW_SERVER, DISABLE_JIP)
+
+		// dac cong
+		FNC_CONSTRUCTOR(vn_mf_fnc_capture_player, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_ctf_handle_flag_height_change, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_ctf_broadcast_notify_immediate, ALLOW_SERVER, DISABLE_JIP)
+
+		// light attachments
+		FNC_CONSTRUCTOR(vn_mf_fnc_attachments_server_attach_chemlight, ALLOW_SERVER, DISABLE_JIP)
+		FNC_CONSTRUCTOR(vn_mf_fnc_attachments_server_attach_flashlight, ALLOW_SERVER, DISABLE_JIP)
+
+	};
+
+	class Commands
+	{
+		mode = 1;
+		ENABLE_JIP;
+
+		// needed during player init process to silence radio messages from
+		// player's character
+		FNC_CONSTRUCTOR(setSpeaker, ALLOW_SERVER, DISABLE_JIP)
+		// fn_operate_wrench.sqf
+		FNC_CONSTRUCTOR(setDamage, ALLOW_SERVER, DISABLE_JIP)
```

Comment:
!todo: needs refactor to not allow `setDamage` and to safely verify the provided object is a vehicle.

Reply:
this needs to happen over in paradigm

Reply:
https://github.com/Bro-Nation/Paradigm/pull/43